packages/devextreme/testing/tests/DevExpress.ui.widgets/loadPanel.tests.js

```diff
@@ -59,8 +60,38 @@ QUnit.module('init', {
         assert.equal($content.find(MESSAGE_SELECTOR).text(), 'Test Loading Message');
     });
 
+    QUnit.test('correct wrapper aria attributes load when showIndicator = false', function(assert) {
+        const label = messageLocalization.format('Loading');
+        const instance = $('#loadPanel').dxLoadPanel({ visible: true, showIndicator: false }).dxLoadPanel('instance');
+        instance.option({ message: '' });
+
+        assert.strictEqual(instance.$wrapper().attr('role'), 'alert');
+        assert.strictEqual(instance.$wrapper().attr('aria-label'), label);
+    });
+
+    QUnit.test('wrapper aria attributes must not load when showIndicator = true', function(assert) {
+        const instance = $('#loadPanel').dxLoadPanel({ visible: true, showIndicator: false });
```

Comment:
It's not an instance, it's a LoadPanel root jquery element. Let's update the naming
```suggestion
        const $loadPanel = $('#loadPanel').dxLoadPanel({ visible: true, showIndicator: false });
```